Added conceptual article for About assigments. (#25049)
github/docs: content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-assignments-in-github-classroom.md

---
title: About assignments in GitHub Classroom

> intro: 'Assignment in {% data variables.product.prodname_classroom %} is a coursework created to be completed by an individual student or a group of students either for grading purposes or for practice.'

Author: ```suggestion
intro: 'Assignment in {% data variables.product.prodname_classroom %} is a coursework created to be completed by an individual student or a group of students either for grading purposes or for practice.'
shortTitle: About assignments
```

## Types of assignments 

You can either create a group assignment or n individual assignment in {% data variables.product.prodname_classroom %}.

### Individual assignments

An individual assignment is coursework for each student who participates in a course on GitHub Classroom. The student is intended to complete an individual assignment without help from other students.

When a student accepts an assignment, GitHub Classroom automatically creates a new repository for the student. 

For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment)."

### Group assignments

A group assignment is collaborative coursework for groups of students on GitHub Classroom. Students can work together on a group assignment in a shared repository, like a team of professional developers.

When a student accepts a group assignment, the student can create a new team or join an existing team. GitHub Classroom saves the teams for an assignment as a set. You can name the set of teams for a specific assignment when you create the assignment, and you can reuse that set of teams for a later assignment.

> For more information, see "[AUTOTITLE](education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment)."

Author: ```suggestion
For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment)."
```

---

> title: About assignments in GitHub Classroom

Reviewer: OlayinkaAtobiloye:assignments

If you integrate your {% data variables.product.prodname_classroom %} with an IDE, students can make changes to their assignment repositories through the IDE, or by checking out and running code locally with the appropriate software. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."

You can reuse existing assignments in any other classroom you have admin access to, including classrooms in a different organization. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/reuse-an-assignment)."

You must create a classroom before you can create an assignment. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms#creating-a-classroom).

## Types of assignments 

You can either create a group assignment or n individual assignment in {% data variables.product.prodname_classroom %}.

### Individual assignments

> An individual assignment is coursework for each student who participates in a course on GitHub Classroom. The student is intended to complete an individual assignment without help from other students.

Reviewer 2: ```suggestion
{% data reusables.classroom.assignments-individual-definition %}
```

Small suggestion to use the reusable here

> When a student accepts an assignment, GitHub Classroom automatically creates a new repository for the student. 

Reviewer 2: ```suggestion
{% data reusables.classroom.classroom-creates-individual-repositories %}
```

Suggestion to instead use the relevant reusable here since we're deleting that paragraph in the intro to better separate content on the different assignment types.

For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment)."

### Group assignments

> A group assignment is collaborative coursework for groups of students on GitHub Classroom. Students can work together on a group assignment in a shared repository, like a team of professional developers.

Reviewer 2: ```suggestion
{% data reusables.classroom.assignments-group-definition %} Students can work together on a group assignment in a shared repository, like a team of professional developers.
```

Suggestion to use the reusable

> When a student accepts a group assignment, the student can create a new team or join an existing team. GitHub Classroom saves the teams for an assignment as a set. You can name the set of teams for a specific assignment when you create the assignment, and you can reuse that set of teams for a later assignment.

Reviewer 2: ```suggestion
When a student accepts a group assignment, the student can create a new team or join an existing team. {% data variables.product.prodname_classroom %} saves the teams for an assignment as a set. You can name the set of teams for a specific assignment when you create the assignment, and you can reuse that set of teams for a later assignment.
```

I'd suggest using that variable, then also turning this into a reusable in `data/reusables/classroom`. It can then be used in the "[About group assignments](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment#about-group-assignments)" section of "[Create a group assignment](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment)."

For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment)."

## Further reading

> 

Reviewer 2: ```suggestion

- "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/use-the-git-and-github-starter-assignment)"
```

I think linking to this starter assignment article as further reading would be helpful too!

> - "[AUTOTITLE](/education/manage-coursework-with-github-classroom/create-an-individual-assignment)."

Reviewer 2: ```suggestion
- "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment)."
```

Updating the path to avoid redirects
